### DISCUSSION TITLE
DKIM signing with RSpamd

### DISCUSSION THREAD

**Discussion by alexanderadam** [docker-mailserver/docker-mailserver#3207] [Q&A]
I followed [the documentation](https://docker-mailserver.github.io/docker-mailserver/edge/config/security/rspamd/#dkim-signing) but outgoing emails are still not signed.
I'm not seeing errors in the log so what's the best way to debug this?

The email headers are missing.

**EDIT:** created [an issue](https://github.com/docker-mailserver/docker-mailserver/issues/3220#issue-1654679064)

**Comment by polarathene** [comment]
@georglauterbach might be able to assist.

**Comment by georglauterbach** [comment]
Please open a dedicated issue and share your configuration; this makes debugging easier :)
